style.css

```diff
@@ -0,0 +1,132 @@
+body {
+  margin: 0 auto;
+}
+
+.nav-section {
+  background-color: #3c3a39;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  padding: 30px 25px 0 10px;
+}
+
+.logo {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  float: left;
+}
+
+.setting-mobile {
+  float: right;
+}
+
+nav {
+  width: 100%;
+  min-height: 48px;
+}
+
+.text-white {
+  color: #fff;
+}
+
+a {
+  text-decoration: none;
+}
+
+.bg-primary {
+  display: flex;
+  background-color: #1c1a19;
+  max-width: 100vw;
+  height: 100vh;
+  padding: 120px 24px;
+  background-image: url(images/bg-image.svg);
+  background-repeat: no-repeat;
+  background-position: right;
+  background-position-y: 60%;
+}
+
+.text-primary {
+  color: #ff6b00;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 56px;
+}
+
+.text-supportive {
+  color: #fff5e1;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+}
+
+.text-description {
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.social-media-section {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+}
+
+.social-media {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 50%;
+  padding: 0;
+}
+
+.social-media > li {
+  list-style: none;
+}
+
+@media screen and (min-width: 768px) {
+  .setting-mobile {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 430px) {
+  .bg-primary {
+    background-size: 25%;
+    background-position-y: 70%;
+  }
+}
+
+@media screen and (max-width: 375px) {
+  .bg-primary {
+    padding: 5%;
+    background-size: 20%;
+    background-position-y: 25%;
+  }
```

Comment:
- [ ] Why are you using media queries now? Please remember you are working on the mobile version of this project so make sure you remove all @media queries and make the design of your mobile version (font-size, font-weight, and text-alignment) look exactly as the mobile version of the Figma design. :pray: 

  | Your Design |Figma Design |
  | :---:   | :---: | 
  | ![image](https://user-images.githubusercontent.com/92303891/234829500-05861224-3d7e-4a05-9641-912593779ef6.png)| ![image](https://user-images.githubusercontent.com/92303891/234829774-216c335f-1ab7-4fc6-890c-87f84677ab3a.png)|

Reply:
@Owusu-Desmond Firstly, thanks for your review. I wanted to clear up a doubt I have about texts, should I write the same text as written on the Figma template or is putting my name instead of "Alex" fine?

Reply:
Hi @rubydevi, when it comes to the text content, it's recommended to use your own information instead of the placeholder text in the Figma template. You can replace "Alex" with your own name and update the other details with your own personal information to make the project truly yours. Let me know if you have any other questions or concerns. 🎯